tests/fixtures.py

```diff
@@ -171,3 +207,56 @@ def finalizer():
             class_instance.pod_obj.delete()
 
     request.addfinalizer(finalizer)
+
+
+@pytest.fixture()
+def create_pvcs(request):
+    """
+    Create multiple PVCs
+    """
+    class_instance = request.node.cls
+
+    def finalizer():
+        """
+        Delete multiple PVCs
+        """
+        if hasattr(class_instance, 'pvc_objs'):
+            [pvc.delete() for pvc in class_instance.pvc_objs]
+
+    request.addfinalizer(finalizer)
+
+    class_instance.namespace = getattr(class_instance, 'namespace', defaults.ROOK_CLUSTER_NAMESPACE)
+
+    class_instance.pvc_objs = helpers.create_multiple_pvc(
+        class_instance.sc_obj.name, class_instance.num_of_pvcs, size=class_instance.pvc_size,
+        namespace=class_instance.namespace
+    )
+
+
+@pytest.fixture()
+def create_pods(request):
+    """
+    Create multiple pods
+    """
+    class_instance = request.node.cls
+
+    def finalizer():
+        """
+        Delete multiple pods
+        """
+        if hasattr(class_instance, 'pod_objs'):
+            [pod.delete() for pod in class_instance.pod_objs]
```

Comment:
I think list is not required here as it is not used for any validation.

Reply:
fixed